packages/form-js-viewer/src/render/components/form-fields/Datetime.js

```diff
@@ -0,0 +1,170 @@
+
+  const isValidDate = (date) => date && !isNaN(date.getTime());
+  const isValidTime = (time) => !isNaN(parseInt(time));
+
+  const useDatePicker = useMemo(() => subtype === DATETIME_SUBTYPES.DATE || subtype === DATETIME_SUBTYPES.DATETIME, [ subtype ]);
+  const useTimePicker = useMemo(() => subtype === DATETIME_SUBTYPES.TIME || subtype === DATETIME_SUBTYPES.DATETIME, [ subtype ]);
+
+  const [ date, time ] = useMemo(() => {
+    switch (subtype) {
+    case DATETIME_SUBTYPES.DATE:
+      return [ new Date(Date.parse(value)), null ];
+    case DATETIME_SUBTYPES.TIME:
+      return [ null, parseIsoTime(value) ];
+    case DATETIME_SUBTYPES.DATETIME: {
+
+      // ensure enough information is provided in the input, as Date.parse() is way too forgiving
+      const isInputSufficient = isDateTimeInputInformationSufficient(value);
+
+      let date, time;
+
+      if (isInputSufficient) {
+        date = new Date(Date.parse(value));
+        time = isValidDate(date) ? 60 * date.getHours() + date.getMinutes() : null;
+      }
+      else {
+        date = getNullDate();
+        time = null;
+      }
+
+      setDateCache(date);
+      setTimeCache(time);
+
+      return [ date, time ];
+    }}
+  }, [ subtype, value ]);
+
+  const evaluate = useCallback(({ time, date }) => {
+
+    let newDateTimeValue = null;
+
+    if (subtype === DATETIME_SUBTYPES.DATE && isValidDate(date)) {
+      newDateTimeValue = serializeDate(date);
+    }
+    else if (subtype === DATETIME_SUBTYPES.TIME && isValidTime(time)) {
+      newDateTimeValue = serializeTime(time, new Date().getTimezoneOffset(), timeSerializingFormat);
+    }
+    else if (subtype === DATETIME_SUBTYPES.DATETIME && isValidDate(date) && isValidTime(time)) {
+      newDateTimeValue = serializeDateTime(date, time, timeSerializingFormat);
+    }
+
+    onChange({ value: newDateTimeValue, field });
+
+  }, [ field, onChange, subtype, timeSerializingFormat ]);
+
+  const setDate = useCallback((date) => {
+    evaluate({ date, time: timeCache });
+    setDateCache(date);
+  }, [ evaluate, timeCache ]);
+
+  const setTime = useCallback((time) => {
+    evaluate({ time, date: dateCache });
+    setTimeCache(time);
+  }, [ evaluate, dateCache ]);
+
+  const allErrors = useMemo(() => {
+
+    // If only one of the two fields is set
+    const shouldCompleteSecondField = !required && (subtype === DATETIME_SUBTYPES.DATETIME) && ((isValidDate(dateCache) && timeCache === null) || (!isValidDate(dateCache) && timeCache !== null));
+    return shouldCompleteSecondField ? [ 'Date and time must both be entered.', ...errors ] : errors;
+
+  }, [ required, subtype, dateCache, timeCache, errors ]);
+
+  const datePickerProps = {
+    id,
+    formId,
+    disabled,
+    disallowPassedDates,
+    date,
+    setDate
+  };
+
+  const timePickerProps = {
+    id,
+    formId,
+    disabled,
+    use24h,
+    timeInterval,
+    time,
+    setTime
+  };
+
+  return <div class={ formFieldClasses(type, allErrors) }>
```

Comment:
When placing two date time pickers below each other, the input element is always on top of the other's dropdown list.

![image](https://user-images.githubusercontent.com/9433996/199238777-f3383504-e493-45bc-91c5-09a3cc802f87.png)